weblate/vcs/git.py

```diff
@@ -564,6 +595,24 @@ def get_remote_configure(
             return (('remote "gerrit"', "url", push_url),)
         return (('remote "gerrit"', "url", None),)
 
+    def configure_remote(
+        self, pull_url: str, push_url: str, branch: str, fast: bool = True
+    ) -> None:
+        """
+        Configure remote repository.
+
+        Gets the gerrit username from push URL and
+        sets it as the value of gitreview.username.
+        """
+        gerrit_user = ""
+        if push_url is not None:
+            if push_url.startswith("ssh://"):
+                gerrit_user = push_url.split("@")[0].split("//")[-1]
+            elif push_url.startswith("git@") or push_url.startswith("https://"):
+                _, _, gerrit_user, _ = self.parse_repo_url(push_url)
```

Comment:
I don't think that owner as returned by parse_repo_url is what you want here - it is owner of the repo at GitHub (for `https://github.com/WeblateOrg/weblate.git` it would be `WeblateOrg`).

Reply:
Hi @nijel, I updated how it was implemented. I also added test with different URLs.